tests/_internal/test_config_header_file.py

```diff
@@ -1,58 +0,0 @@
-#
-# Copyright (C) 2020 Arm Mbed. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-#
-import pathlib
```

Comment:
Shouldn't the config_header_file tests be removed in the same commit that removes this capability? Otherwise, how does each commit individually pass tests?

Reply:
We're squashing commits at the moment so "the PR' passes the tests.

Reply:
:/